app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def like_count
+    post.increment!(:likes_count)
+  end
```

Comment:
- [ ] Use the existing attribute `LikesCounter` instead of `likes_count`. The above implementation will output the `unknown attribute 'likes_count' for Like`